src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -162,6 +161,7 @@ export interface SavedObjectsTableState {
   isIncludeReferencesDeepChecked: boolean;
   currentWorkspace?: WorkspaceObject;
   workspaceEnabled: boolean;
+  workspaceIdNameMap: Map<string, string>;
```

Comment:
Shall we call it:
```suggestion
  workspaceNameIdMap: Map<string, string>;
```
Because it's name->id map